android/WorkshopSkeleton/app/src/main/java/com/mobiledgex/workshopskeleton/FaceProcessorActivity.java

```diff
@@ -57,4 +73,39 @@ public void finish() {
         super.finish();
     }
 
+    public Socket exampleDeveloperWorkflow() {
+        ////////////////////////////////////////////////////////////////////////////////////////////
+        // TODO: Copy/paste the code to follow example developer workflow and get FaceDetection started.
+        return null;
+        ////////////////////////////////////////////////////////////////////////////////////////////
+    }
+
+    private void exampleDevloperWorkflowBackground() {
```

Comment:
We should migrate away from AsyncTask as an API since it is going to be deprecated with other options available. But this is fine for now.